contracts/interfaces/IERC20Extended.sol

```diff
@@ -0,0 +1,25 @@
+pragma solidity ^0.6.0;
+
+import "@uniswap/v2-periphery/contracts/interfaces/IERC20.sol";
+
+interface IERC20Extended is IERC20 {
+    function transferExactDest(address _to, uint256 _value) external returns (bool success);
+
+    function transferExactDestFrom(
+        address _from,
+        address _to,
+        uint256 _value
+    ) external returns (bool success);
+
+    function getReceivedAmount(
```

Comment:
this name seems confusing.
I know we discussed.
but when looking back on this after a few weeks it seems misleading.
Received is past tense and used to query something that has happened. I suggest we discuss this name again.

Reply:
`Received` is passive form which indicates the `to` address will  be received that amount